Sources/SwiftFormat/Rules/NeverForceUnwrap.swift

```diff
@@ -16,6 +16,7 @@ import SwiftSyntax
 ///
 /// This rule does not apply to test code, defined as code which:
 ///   * Contains the line `import XCTest`
+///   * The function is marked with `@Test` attribute
```

Comment:
Is this noise from another PR that was merged in?

Reply:
> Is this noise from another PR that was merged in?

This was because I ran `generate-swift-format`, and there have been some manual edits to the generated files. So I made the code edits necessary to keep the generated file the same.